subsys/net/lib/lwm2m/Kconfig

```diff
@@ -240,6 +240,14 @@ config LWM2M_DEVICE_ERROR_CODE_MAX
 	  This value sets the maximum number of error codes that the device
 	  object will store before ignoring new values.
 
+config LWM2M_DEVICE_EXT_DEV_INFO_MAX
+	int "Maximum # of device obj external device info to store"
+	default 5
+	range 1 20
```

Comment:
Is the range coming from some spec, or are the values here just selected to be suitable common values?

Reply:
Spec does not define maximum number of multiinstance resources, I've just used some arbitrary values as other multiinstance resources above.